src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -135,11 +141,17 @@ public void killConnection(String key) {
    * Requests a token from the key server.
    */
   public String requestTokenFromServer() {
+    //TODO: Replace when the StunFileGeneration is ready
+    pathToStunDumpFile = this.getClass().getClassLoader().getResource("ice4jDemoDump.ice")
+        .toExternalForm().replace("file:","");
+
     keyServerSetup();
 
-    String command = commandStringGenerator(pathToResourceDocker, "start_x11vnc.sh");
+    String command = commandStringGenerator(
+        pathToResourceDocker, "start_x11vnc.sh", pathToStunDumpFile);
     String key = systemCommander.executeTerminalCommand(command);
     this.key.set(key); // update key in model
+    System.out.println(getKey());
```

Comment:
And i added a setter for the Key, I think this is appropriate, as also the one connecting will set this key.